test/rust_analyzer/rust_analyzer_test_runner.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Runs Bazel build commands over rustfmt rules, where some are expected
```

Comment:
I think this mentions rustfmt incorrectly?

Reply:
It does, I'll fix later this afternoon